### PR TITLE
Improve Swagger doc

### LINKED ISSUE #302: Fix Swagger tags
Try to categorize properly services in Swagger, try to keep only one tag per service

### PR DESCRIPTION
**Pull request by smadbe**
* Use a single swagger tag for service to prevent issues
* Shorten summaries which were too long to be summaries (to fit in the menu and to be easy to browse)

Fixes #302

### CODE REVIEW

File: app/api/auth/login_callback.go [hunk 1]
```diff
@@ -18,8 +18,10 @@ import (
 
 // swagger:operation GET /auth/login-callback auth userLoginCallback
 // ---
-// summary: Callback to which the user is redirected after authentication with the login module
-// description: Creates or updates the authenticated user in the DB using the data returned by the login module,
+// summary: User login callback
+// description: Callback to which the user is redirected after authentication with the login module
+//
```
Comment (zenovich) [inline app/api/auth/login_callback.go]: One blank line is not enough to make a new paragraph

File: app/api/auth/login.go [hunk 2]
```diff
@@ -13,8 +13,10 @@ import (
 
 // swagger:operation POST /auth/login auth authLogin
 // ---
-// summary: Redirect with appropriate security parameters to the login module so that the user can log in
-// description: Generates a state value (a random 32-char [0-9a-z] string) and a random cookie (a random 32-char [0-9a-z] string).
+// summary: User login (redirect)
+// description: Redirect with appropriate security parameters to the login module so that the user can log in.
+//
```
Comment (zenovich) [inline app/api/auth/login.go]: One blank line is not enough to make a new paragraph.

File: app/api/contests/get_administered_list.go [hunk 3]
```diff
@@ -35,8 +35,9 @@ type contestAdminListRow struct {
 
 // swagger:operation GET /contests/administered contests contestAdminList
 // ---
-// summary: Get the contests that the user has administration rights on
-// description: >
+// summary: List administered contests
+// description:   Get the contests that the user has administration rights on
+//
```
Comment (zenovich) [inline app/api/contests/get_administered_list.go]: The same here

**Comment by smadbe** [comment]
I thought I had fix it all. Thanks for noticing. Fixed.

**Pull request merged**